### DISCUSSION TITLE
Tailwind does not allow ReactMarkdown to work properly

### DISCUSSION THREAD

**Discussion by AlexandrKudryavtsev** [tailwindlabs/tailwindcss#9276] [Help]
I am developing an application on Tailwind. It should have functionality for displaying markdown articles. Initially I wanted to use ReactMarkdown for rendering, but it does not work correctly with Tailwind.

- No headers
- No listings
- No tables

Here is an example:

```
import React from 'react';
import ReactMarkdown from 'react-markdown';
import remarkGfm from 'remark-gfm';

const markdown = `
# Dillinger
## _The Last Markdown Editor, Ever_

[![N|Solid](https://cldup.com/dTxpPi9lDf.thumb.png)](https://nodesource.com/products/nsolid)

[![Build Status](https://travis-ci.org/joemccann/dillinger.svg?branch=master)](https://travis-ci.org/joemccann/dillinger)

Dillinger is a cloud-enabled, mobile-ready, offline-storage compatible,
AngularJS-powered HTML5 Markdown editor.

- Type some Markdown on the left
- See HTML in the right
- ✨Magic ✨

## Features

- Import a HTML file and watch it magically convert to Markdown
- Drag and drop images (requires your Dropbox account be linked)
- Import and save files from GitHub, Dropbox, Google Drive and One Drive
- Drag and drop markdown and HTML files into Dillinger
- Export documents as Markdown, HTML and PDF
`;

function ArticlePage() {
  return <ReactMarkdown children={markdown} remarkPlugins={[remarkGfm]} className='prose prose-lg'/>;
}

export default ArticlePage;
```
On the first screen there is an example with Tailwind, and on the second - without it.

![image](https://user-images.githubusercontent.com/66219061/188709846-1e5ff170-0367-433d-a5ec-e81aba304636.png)
![image](https://user-images.githubusercontent.com/66219061/188709908-ccdfa67e-e969-4ba2-a350-646ed1756347.png)

**Comment by wongjn** [comment]
It seems like the `prose` and `prose-lg` classes might not be generated by Tailwind. I see that these classes exist in your supplied source code, but it seems like the file this source code sits in is not covered by the `content` globs in your Tailwind config.
